diff --git a/src/webpack.config.js b/src/webpack.config.js
index 2fa005bffa952dcdbe5ff657d54c9f090f60f1d0..5dfe3e872f077e440bf903d038e8a4a77039e24f 100644
--- a/src/webpack.config.js
+++ b/src/webpack.config.js
@@ -123,14 +123,6 @@ const webpackConfig = {
                             },
                             importLoaders: 1
                         }
-                    },
-                    {
-                        loader: 'postcss-loader',
-                        options: {
-                            config: {
-                                path: path.join(__dirname, 'postcss.config.js')
-                            }
-                        }
                     }
                 ]
             },
